Aptoide/aptoide-client-v8
# [ASV-1592] Remove unity ads initialization

**What does this PR do?**
This PR aims at fixing unity ads initialization. Since unity ads is a certified network we don't need to initialize it. This was causing a problem where the ad would be shown, but the unity ads impression was not being sent.

**Database changed?**

 No

**Where should the reviewer start?**

- [x] MainActivity.java

**How should this be manually tested?**
You can check for show attempt and success show logs. Since there are a lot of logs and we are not sure about the correct message, i think you can just debug the sdk and add breakpoints on UnityInterstitial class, in the onUnityAdsStart method where you have the SHOW_SUCCESS message being logged.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1592](https://aptoide.atlassian.net/browse/ASV-1592)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
